src/CGIHandler.cpp

```diff
@@ -8,11 +8,18 @@ CGIHandler::~CGIHandler()
 {
 }
 
+CGIHandler::CGIHandler(Connection *conn)
+{
+	_conn = conn;
+}
+
```

Comment:
Is this supposed to be like this?

Reply:
I added a note somewhere else, it would be better not to have it as a pointer, but I couldn't solve the cross declarations problems...